zyte_common_items/pages/__init__.py

```diff
@@ -38,3 +38,53 @@
     BaseSocialMediaPostPage,
     SocialMediaPostPage,
 )
+
+__all__ = [
+    "ArticleListPage",
+    "ArticleNavigationPage",
+    "ArticlePage",
+    "AutoArticleListPage",
+    "AutoArticleNavigationPage",
+    "AutoArticlePage",
+    "AutoBusinessPlacePage",
+    "AutoForumThreadPage",
+    "AutoJobPostingNavigationPage",
+    "AutoJobPostingPage",
+    "AutoProductListPage",
+    "AutoProductNavigationPage",
+    "AutoProductPage",
+    "AutoRealEstatePage",
+    "AutoSerpPage",
+    "AutoSocialMediaPostPage",
+    "BaseArticleListPage",
+    "BaseArticleNavigationPage",
+    "BaseArticlePage",
+    "BaseBusinessPlacePage",
+    "BaseForumThreadPage",
+    "BaseJobPostingNavigationPage",
+    "BaseJobPostingPage",
+    "BasePage",
+    "BaseProductListPage",
+    "BaseProductNavigationPage",
+    "BaseProductPage",
+    "BaseRealEstatePage",
+    "BaseSearchRequestTemplatePage",
+    "BaseSerpPage",
+    "BaseSocialMediaPostPage",
+    "BusinessPlacePage",
+    "DescriptionMixin",
+    "ForumThreadPage",
+    "HasMetadata",
+    "JobPostingNavigationPage",
+    "JobPostingPage",
+    # "MetadataT",
```

Comment:
Not sure what to do with this (it was moved from `.pages.mixins` to `.components.metadata` in #92), we could drop it outright or deprecate this import first.